src/pages/api/account/setDetails.ts

```diff
@@ -0,0 +1,78 @@
+import { db } from '@utils/prisma';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getSession } from 'next-auth/react';
+
+/**
+ * Changes user settings
+ * @param setProperties an array of key value pairs of user fields to be updated. Their values need to be strings for now, but maybe in the futrure we can transfer them via JSON
+ * @param req the HTTP request
+ */
+export const setUserDetails = async (
+  setProperties: { [key: string]: string },
+  req: NextApiRequest,
+) => {
+  const session = await getSession({ req });
+
+  if (!session?.user?.email) {
+    // eslint-disable-next-line prettier/prettier
```

Comment:
🤔

Reply:
I have no idea how to fix this, prettier and eslint both seem to have different rules for using `''` when there is a `'` in the string

Reply:
ok solved 😄